ru/turing/courses/lesson4/Gender.java

```diff
@@ -1,43 +1,52 @@
 package ru.turing.courses.lesson4;
 
-/**
- * Пол человека
- *
- * @author diagorn
- */
 public enum Gender {
-    MALE("Male", 'M', "Мужской пол"),
-    FEMALE("Female", 'F', "Женский пол");
+    MALE("MALE", 'M', "Мужской пол"),
+    FEMALE("FEMALE", 'F', "Женский пол");
```

Comment:
А это что за еврейские фокусы? Верни к "Male" и "Female"
```suggestion
    FEMALE("Female", 'F', "Женский пол");
```